### DISCUSSION TITLE
Spacers

### DISCUSSION THREAD

**Discussion by m0ckery** [sanni/cartreader#218] [Need help]
Just a quick question - what sized spacers/standoffs are suggested for assembly? The assembly video suggests M3x15 but the screws are M2? I'm not seeing the spacers suggested on the BoM - Thanks!

**Comment by sanni** [comment]
Most of the case designs have the spacers integrated into the 3D print with holes for M2 screws.
If the spacer is not part of the case design I would go with M2x15 since M3x15 can block the GBA slot.
